play-json-ops-common-213/src/main/scala/play/api/libs/json/ops/v4/ClassNameUtils.scala

```diff
@@ -0,0 +1,21 @@
+package play.api.libs.json.ops.v4
+
+private[v4] object ClassNameUtils {
+
+  /**
+   * Similar to the Class.getSimpleName method, except it does not throw any exceptions and
+   * handles Scala inner classes better.
+   */
+  def safeSimpleClassName(cls: Class[_]): String = {
```

Comment:
I'm a little afraid to change the behavior on this in this PR. I am mostly trying to upgrade everything to the latest versions so that I can use the latest sbt to publish to Sonatype.